tests/foreman/cli/test_docker.py

```diff
@@ -1249,7 +1249,7 @@ def test_positive_pull_image(self):
             # publishing takes few seconds sometimes
             result, _ = wait_for(
                 lambda: ssh.command(
-                    'docker pull {0}'.format(repo['published-at']),
+                    'docker pull {}'.format(repo['published-at']),
```

Comment:
these are throughout this file
```suggestion
                    f'docker pull {repo["published-at"]}',
```